index.js

```diff
@@ -24,6 +24,7 @@ const UtilityToken = require('./src/ContractInteract/UtilityToken');
 const Mosaic = require('./src/Mosaic');
 
 const Utils = require('./src/utils/Utils');
+const ProofGenerator = require('./src/utils/Proof');
```

Comment:
Why is it called `ProofGenerator` in the export, but only `Proof` as a class. Should those not match? For example to find the documentation once we host the JSDocs?